msal/src/Microsoft.Identity.Client/AuthenticationResult.cs

```diff
@@ -58,12 +60,12 @@ internal AuthenticationResult(MsalAccessTokenCacheItem msalAccessTokenCacheItem,
         }
 
         /// <summary>
-        /// Gets the Access Token requested.
+        /// Gets the requested Access Token.
         /// </summary>
         public virtual string AccessToken => _msalAccessTokenCacheItem.Secret;
 
         /// <summary>
-        /// Gets the Unique Id of the user.
+        /// Gets the Unique Id of the account.
         /// </summary>
```

Comment:
>/// </summary> [](start = 8, length = 14)

should perhaps also say it will be null if the value is not found?

Reply:
done